src/dropdown/dropdown.scss

```diff
@@ -236,18 +236,18 @@ $height-small: 30px !default;
       position: relative;
     }
 
-    .Select-control .Select-value {
+    &.has-value .Select-control .Select-value {
```

Comment:
`.Select-value` exists only if the `.dropdown` has `.has-value`, then I'm sure no regression will be introduced by this change